# HG changeset patch
# Parent 49313c73f095b8f38fcd9e460c14956917077683
localmod: failed chunks from X86InstrInfo.h is moved into X86BaseInfo.h

diff -r 49313c73f095 -r d628685eae13 lib/Target/X86/MCTargetDesc/X86BaseInfo.h
--- a/lib/Target/X86/MCTargetDesc/X86BaseInfo.h	Tue Aug 09 10:58:04 2011 -0700
+++ b/lib/Target/X86/MCTargetDesc/X86BaseInfo.h	Tue Aug 09 13:16:42 2011 -0700
@@ -248,6 +248,7 @@
     /// manual, this operand is described as pntr16:32 and pntr16:16
     RawFrmImm16 = 44,
 
+    CustomFrm      = 62, // @LOCALMOD
     FormMask       = 63,
 
     //===------------------------------------------------------------------===//
@@ -469,6 +470,7 @@
     case X86II::MRMSrcReg:
     case X86II::RawFrmImm8:
     case X86II::RawFrmImm16:
+    case X86II::CustomFrm: // @LOCALMOD
        return -1;
     case X86II::MRMDestMem:
       return 0;
